test/liquidation/liquidation-bot-script.ts

```diff
@@ -7,38 +7,53 @@ describe('Liquidation Bot', function () {
   after(resetHardhatNetwork);
 
   describe('liquidateUnderwaterBorrowers', function () {
-    it('liquidates underwater borrowers', async function () {
-      const {
-        comet,
-        liquidator,
-        users: [signer, underwater],
-        assets: { dai, usdc },
-        whales: { usdcWhale }
-      } = await makeLiquidatableProtocol();
-
-      // transfer USDC to comet, so it has money to pay out withdraw to underwater user
-      await usdc.connect(usdcWhale).transfer(comet.address, 300000000n); // 300e6
-      await dai.connect(underwater).approve(comet.address, 120000000000000000000n);
-      await comet.connect(underwater).supply(dai.address, 120000000000000000000n);
-      // withdraw to ensure that there is a Withdraw event for the user
-      await comet.connect(underwater).withdraw(usdc.address, 10e6);
-      // put the position underwater
-      await comet.setBasePrincipal(underwater.address, -(exp(200, 6)));
-
-      expect(await comet.isLiquidatable(underwater.address)).to.be.true;
-
-      await liquidateUnderwaterBorrowers(
-        comet,
-        liquidator,
-        {signer},
-        'mainnet'
-      );
-
-      expect(await comet.isLiquidatable(underwater.address)).to.be.false;
-
-      const assetAddresses = await getAssets(comet);
-      expect(await hasPurchaseableCollateral(comet, assetAddresses, 1)).to.be.false;
-    });
+    const assetAmounts = {
+      'comp': exp(120, 18),
+      'link': exp(200, 18), // XXX small amount; increase transfer to user
+      'uni': exp(200, 18), // XXX small amount; increase transfer to user
```

Comment:
Are these still XXX?

Reply:
yep, they're still todo. but it's not urgent that we update it in this PR; we do much larger swaps with UNI and LINK in the scenarios and the Forge tests